mysql-test/mroonga/include/mroonga/have_mroonga.inc

```diff
@@ -44,4 +44,10 @@ if ($have_default_tmp_storage_engine_variable) {
   set default_tmp_storage_engine=Mroonga;
 }
 
+if ($mariadb) {
+  if ($version_10_10_or_later) {
+    SET explicit_defaults_for_timestamp=false;
```

Comment:
The default value of `explicit_defaults_for_timestamp` is changed false to true since 10.10 in MariaDB.
Some tests fail if `explicit_defaults_for_timestamp` is true because a displayed type is different from the expected results.
In order to avoid that failing, `explicit_defaults_for_timestamp` are set to false.

But I doubt that it is good to set `explicit_defaults_for_timestamp=false` in this file.
The reason why I added it in this file is because this file is executed on all test cases, 
but I feel it is not good in semantic.
(`have_mroonga` should do about Mroonga, but `explicit_defaults_for_timestamp` is the parameter for DBMS, not Mroonga)

Is there any good place to set `explicit_defaults_for_timestamp` to false?

Reply:
How about mtr's option as below.

https://github.com/mroonga/mroonga/blob/master/packages/yum/test.sh#L191

```
--mysqld=--explicit_defaults_for_timestamp=ON
```

BTW, I think that we should `explicit_defaults_for_timestamp` is ON.
Because TIMESTAMP with implicit DEFAULT value is deprecated.
Therefore, in generally, we should set the value of `explicit_defaults_for_timestamp` is ON.

The default value of `explicit_defaults_for_timestamp` in MySQL 8.0.
https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_explicit_defaults_for_timestamp

Reply:
> Are these 3 tests depend on the default value? 

No, just not specifying DEFAULT.

> How about just adding `DEFAULT CURRENT_TIMESTAMP` explicitly to these 3 tests?

So, I have adopted this suggest.